zpi-backend/src/main/java/pwr/zpibackend/services/impl/thesis/ReservationService.java

```diff
@@ -258,8 +258,10 @@ private void sendMailDuringReservationCreation(Thesis thesis, Reservation newRes
             mailService.sendHtmlMailMessage(student.getMail(), MailTemplates.RESERVATION_STUDENT,
                     student, null, thesis);
         } else {
-            mailService.sendHtmlMailMessage(student.getMail(),
-                    MailTemplates.RESERVATION_ADMIN, student, null, thesis);
+            if (!(newReservation.getStudent().equals(thesis.getLeader()))) {
```

Comment:
Czy w tym przypadku lider dostanie również maila, że został zapisany przez admina? Jeszcze jest przypadek kiedy to admin zapisuje, a taki nie jest chyba obsługiwany w funkcji `sendMailDuringReservationCreation`